src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/reset_global_properties_request.cc

```diff
@@ -197,8 +197,20 @@ bool ResetGlobalPropertiesRequest::ResetHelpPromt(
     SendResponse(false, mobile_apis::Result::APPLICATION_NOT_REGISTERED);
     return false;
   }
+  const std::vector<std::string>& help_prompt =
+      application_manager_.get_settings().help_prompt();
+
   smart_objects::SmartObject so_help_prompt =
       smart_objects::SmartObject(smart_objects::SmartType_Array);
+  
+  for (uint32_t i = 0; i < help_prompt.size(); ++i) {
+    smart_objects::SmartObject helpPrompt =
```

Comment:
@ValeriiMalkov please use our coding style naming. And better to rename this to `help_prompt_item`

Reply:
@AKalinich-Luxoft That was fixed at the commit 8b18082df6387812f7a92870da98a12c9db9b15e